Remotion/ObjectBinding/Web.Development.WebTesting.TestSite.Shared/Controls/BocListAsGridUserControl.ascx

```diff
@@ -172,4 +174,59 @@
       </td>
       <td>&nbsp; (always invalid)</td>
     </tr>
+  <tr>
+    <td></td>
+    <td>
+      <testsite:TestBocListWithRowMenuItems
+        ID="JobList_Validation"
```

Comment:
`JobList_Validation.SwitchListIntoEditMode();`

is missing. See `OnLoad` method in code behind.